### PR TITLE
REMOVE ngsiv1 ops aligned with CB 4.0.0

### PR DESCRIPTION
**Pull request by fgalan**
Continuation of PR https://github.com/telefonicaid/fiware-pep-steelskin/pull/523

### CODE REVIEW

File: lib/plugins/orionPlugin.js [hunk 1]
```diff
@@ -72,75 +68,6 @@ function translateAction(logger, originalAction) {
     return action;
 }
 
-/**
- * Extract the action from an XML body.
- *
- * @param {Object} logger        Instance of logops.Logger
- * @param {String} body          Raw string payload.
- */
-function inspectBodyXML(logger, body, callback) {
```
Comment (fgalan) [inline lib/plugins/orionPlugin.js:81]: XML in Orion was removed in version 1.0.0 (more than 8 years ago. It's a bit surprising this has survived so long in PEP code :)

NTC (informative)

**Review by AlvaroVega — APPROVED** [review]
LGTM

File: lib/plugins/orionPlugin.js (hunk 1 above)
Comment (AlvaroVega) [reply lib/plugins/orionPlugin.js:81]: Keypass still uses XML, but not usually is exposed and then no needs a pep.

Comment (fgalan) [reply lib/plugins/orionPlugin.js:81]: You are right. I was referring to XML in the NGSI API. The rests of XML support in PEP (Keypass, etc.) has been preserved.

**Pull request merged**